Makefile

```diff
@@ -42,8 +42,13 @@ compose-plugin: ## Compile the compose cli-plugin
 	--output ./bin
 
 .PHONY: e2e-compose
-e2e-compose: ## Run End to end local tests. Set E2E_TEST=TestName to run a single test
-	gotestsum $(TEST_FLAGS) ./pkg/e2e -- -count=1
+e2e-compose: ## Run end to end local tests in plugin mode. Set E2E_TEST=TestName to run a single test
+	go test $(TEST_FLAGS) -count=1 ./pkg/e2e
+
+.PHONY: e2e-compose-standalone
+e2e-compose-standalone: ## Run End to end local tests in standalone mode. Set E2E_TEST=TestName to run a single test
+	go test $(TEST_FLAGS) -count=1 --tags=standalone ./pkg/e2e
+
 
```

Comment:
Should we have a target to do both, e.g.
```
.PHONY: e2e
e2e: e2e-compose e2e-compose-standalone
```
?

Reply:
I started like this... But I don't see who would use it...
In the CI it wouldn't be used neither. Cuz like this it cannot be parallelised.

Reply:
Bu yes... I can add that